README.md

```diff
@@ -1,4 +1,25 @@
 Le site de l'association Agile France
 =====================================
 
-Ce site est purement statique, et peut être édité directement en tant que HTML.
+## Développement
+
+Ce site est construit avec [Jekyll](https://jekyllrb.com/), un générateur de sites statiques. La version utilisée est celle [actuellement en production](https://pages.github.com/versions/) sur GitHub Pages.
+
+Pour l'améliorer, installer [Ruby](https://www.ruby-lang.org/fr/) et [Jekyll](https://jekyllrb.com) dans leurs [versions de production](https://pages.github.com/versions/) :
+
+```sh
+git clone https://github.com/betagouv/beta.gouv.fr.git
+cd beta.gouv.fr
+gem install bundler --no-ri --no-rdoc
+bundle install
```

Comment:
I guess we are missing the Gemfile? `Could not locate Gemfile` (not familiar with Ruby ecosystem)